src/js/lib.js

```diff
@@ -726,16 +709,15 @@ vjs.parseUrl = function(url) {
  * @private
  */
 function _logType(type, args){
-  var argsArray, noop, console;
-
   // convert args to an array to get array functions
-  argsArray = Array.prototype.slice.call(args);
+  let argsArray = Array.prototype.slice.call(args);
   // if there's no console then don't try to output messages
   // they will still be stored in vjs.log.history
   // Was setting these once outside of this function, but containing them
   // in the function makes it easier to test cases where console doesn't exist
-  noop = function(){};
-  console = window['console'] || {
+  let noop = function(){};
```

Comment:
If it's actually improving something that's one thing, but otherwise you're gonna send a bunch of contributors to Google to look up wtf it does. It's not explicit.